plugins/es-repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3Repository.java

```diff
@@ -69,12 +69,10 @@
 
     static final String TYPE = "s3";
 
-    public static List<Setting<?>> mandatorySettings() {
-        return List.of(ACCESS_KEY_SETTING, SECRET_KEY_SETTING);
```

Comment:
Is there documentation we need to update to reflect this change?

Reply:
Good point, I think we should mention the IAM roles here https://crate.io/docs/crate/reference/en/latest/sql/statements/create-repository.html#s3